### PR TITLE
Display Tensorflow version during build and at runtime

### PR DESCRIPTION
**Pull request by phlash**
To help with diagnosis of build and runtime issues - this shows the version of Tensorflow being used in a build (after any forced checkout), and the linked runtime version.

### CODE REVIEW

File: app/deepseg.cc [hunk 1]
```diff
@@ -327,6 +327,7 @@ int main(int argc, char* argv[]) try {
 	printf("%s version %s\n", argv[0], _STR(DEEPSEG_VERSION));
 	printf("(c) 2021 by floe@butterbrot.org & contributors\n");
 	printf("https://github.com/floe/backscrub\n");
+	printf("(running with Tensorflow Lite version: %s)\n", bs_tensorflow_version());
```
Comment (BenBE) [inline app/deepseg.cc:330]: Should display both runtime and build time versions for TFLite.

File: app/deepseg.cc [hunk 2]
```diff
@@ -327,6 +327,7 @@ int main(int argc, char* argv[]) try {
 	printf("%s version %s\n", argv[0], _STR(DEEPSEG_VERSION));
 	printf("(c) 2021 by floe@butterbrot.org & contributors\n");
```
Comment (BenBE) [inline app/deepseg.cc:328]: May update this line.

Comment (phlash) [reply app/deepseg.cc:328]: Yep -that would be tidier :smile:

File: app/deepseg.cc (hunk 1 above)
Comment (phlash) [reply app/deepseg.cc:330]: will do.

File: app/deepseg.cc [hunk 3]
```diff
@@ -324,10 +324,9 @@ std::optional<std::string> resolve_path(const std::string& provided, const std::
 
 int main(int argc, char* argv[]) try {
 
-	printf("%s version %s\n", argv[0], _STR(DEEPSEG_VERSION));
+	printf("%s version %s (Tensorflow build: %s, run-time: %s)\n", argv[0], _STR(DEEPSEG_VERSION), _STR(TF_VERSION), bs_tensorflow_version());
```
Comment (BenBE) [inline app/deepseg.cc:327]:
```suggestion
	printf("%s version %s (Tensorflow: build %s, run-time %s)\n", argv[0], _STR(DEEPSEG_VERSION), _STR(TF_VERSION), bs_tensorflow_version());
```

**Pull request merged**